data/json/uncraft/ammo/40x46mm.json

```diff
@@ -30,7 +30,8 @@
       [ [ "lgpistol_primer", 1 ] ],
       [ [ "chem_compositionb", 32 ] ],
       [ [ "gunpowder_shotgun", 35 ] ],
-      [ [ "impact_fuze", 1 ] ]
+      [ [ "impact_fuze", 1 ] ],
+	  [ [ "40x46mm_m199_casing", 1 ] ]
```

Comment:
This seems to be in a different formatting style, so that's probably why the JSON linter failed. I'm not sure what the JSON linting style is for this codebase but if this needs to be fixed before merge someone who knows what it should look like can fix it.

Reply:
https://dev.narc.ro/cataclysm/format.html
Here's the linter we use.

Reply:
You can fix it too, there's [a web linter](http://dev.narc.ro/cataclysm/format.html), and the binaries you can download have a linter binary as well (though no instructions on use).

In this case, you're using tabs, not spaces.

Reply:
applied the web linter, and created a new commit with linted Json